apps/remix-ide/src/remixAppManager.js

```diff
@@ -16,7 +16,7 @@ const requiredModules = [ // services + layout views + system views
 // dependentModules shouldn't be manually activated (e.g hardhat is activated by remixd)
 const dependentModules = ['foundry', 'hardhat', 'truffle', 'slither']
 
-const loadLocalPlugins = ["doc-gen", "doc-viewer", "etherscan", "vyper"]
+const loadLocalPlugins = ["doc-gen", "doc-viewer", "etherscan", "vyper", "walletconnect"]
```

Comment:
These are actually directory names, not plugin names. And as the plugin name is not the directory name it won't load it. So we either change everything to walletconnect instead of mixing with wallet-connect or change this to the dir or rename this var so it's clear it's a directory.